.github/gh_matrix_builder.py

```diff
@@ -137,19 +137,27 @@ def macos_config(overrides):
     return base_config
 
 
-# common installcheck_args for all pg15 tests
+# common installcheck_args for all scheduled pg15 tests
 # partialize_finalize is ignored due to #4937
 pg15_installcheck_args = "IGNORES='partialize_finalize'"
 
+# common installcheck_args for all non-scheduled pg15 tests (e.g. PRs)
+# partialize_finalize is ignored due to #4937
+# dist_move_chunk and dist_param ignored due to flakiness
+pg15_installcheck_args_pr = "IGNORES='partialize_finalize dist_move_chunk dist_param'"
```

Comment:
I think you could consider:
* add some conditional if this is a pr or not
  * optionally fill up some var with this ignores
* add that to all of the existing `installcheck_args`